src/main/java/appeng/container/implementations/ContainerQNB.java

```diff
@@ -29,14 +29,14 @@
 public class ContainerQNB extends AEBaseContainer
 {
 
-	final TileQuantumBridge quantumBridge;
+	private final TileQuantumBridge quantumBridge;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "quantumBridge" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)